README.md

```diff
@@ -20,7 +22,19 @@ This is driven from the [json data file](src/ss58-registry.json) which contains
 
 ## Process:
 
-If you wish to add an additional account type then please raise a pull request adding to the json file.
+1. Fork and clone this repo
+
+2. Add an additional account type to `ss58-registry.json` (contiguous prefixes are better)
+
+3. Bump the patch (middle) version number of the `Cargo.toml` by running:
```

Comment:
```suggestion
3. Bump the patch (last) version number of the `Cargo.toml` by running:
```

The middle is the minor

Reply:
ok we probably want to be bumping minor then - adding an enum arm is a minor not a patch in my head.